lib/checkbufferoverrun.cpp

```diff
@@ -1836,7 +1836,7 @@ CheckBufferOverrun::ArrayInfo::ArrayInfo(const Variable *var, const SymbolDataba
  * checking should be based on ArrayInfo from the start and then
  * this will not be needed as the declare can be used instead.
  */
-CheckBufferOverrun::ArrayInfo::ArrayInfo(unsigned int id, const std::string &name, MathLib::bigint size1, MathLib::bigint n)
+CheckBufferOverrun::ArrayInfo::ArrayInfo(unsigned int id, const std::string &name, MathLib::biguint size1, MathLib::bigint n)
```

Comment:
I claim that it's not good practice to use 'unsigned' to say some argument is non-negative.